pages/blog/index.tsx

```diff
@@ -47,6 +50,61 @@ const Blog: NextPage<{
           <div className="max-w-xl mx-auto">
             <PostList {...blogPostResponse} />
           </div>
+          {total > 0 && (
+            <nav>
```

Comment:
Can you put it inside a component - `<Pagination/>` instead? Also, it should be inside `PostList` component otherwise, the pagination won't appear in `/blog/tag/:tagSlug` route.